### DISCUSSION TITLE
Is there a native way to open bootstrap-4 collapse element with external anchor link?

### DISCUSSION THREAD

**Discussion by Rektalizer** [twbs/bootstrap#37404] [Q&A]
I am using .ejs templates to run content on my pages. For FAQ I am using Bootstrap collapse elements like this:

```
<p>
    <button class="btn btn-success btn-lg" type="button" data-toggle="collapse" data-target="#questionProject3"
            aria-expanded="false"
            aria-controls="questionProject3">
        Question3
    </button>
</p>
<a class="collapse" id="questionProject3">
    <div class="card card-body">
        Answer3
    </div>
</a>
```

Is there any possible way to open a certain collapsible with external anchor link like this for example: www.mywebsite.com/faq#questionProject3 ?

By native I mean that I do not use jQuery and trying to avoid JS usage in my templates.

**Comment by tbolt** [comment]
There's no native way to achieve that. Having some javascript run on page load is likely the best/only way.
